grabbit/build.gradle

```diff
@@ -101,6 +101,12 @@ configurations.cq_package {
 
     exclude group: 'org.apache.commons', module: 'commons-lang3'
     exclude group: 'org.apache.commons', module: 'commons-lang'
+
+    // 6.x exclude bundles
+    exclude group: 'com.google.guava', module: 'guava'
+    exclude group: 'org.apache.httpcomponents', module: 'httpclient-osgi'
+    exclude group: 'org.apache.httpcomponents', module: 'httpcore-osgi'
+    exclude module: 'cq-workflow-console-5.8.2'
```

Comment:
If you use the proper dependency syntax, you can do the "normal" thing here too:
```groovy
exclude module: 'cq-workflow-console'
```